docs/content/posts/2023-11-20-release-1.16.3.md

```diff
@@ -0,0 +1,137 @@
+---
+title:  "Apache Flink 1.16.3 Release Announcement"
+date: "2023-11-24T00:00:00.000Z"
+aliases:
+- /news/2023/11/24/release-1.16.3.html
```

Comment:
minor: the date is different the date part of the file name

Reply:
Thanks for your review!

Good catch, and updated.